src/main/groovy/liquibase/harness/change/ChangeObjectTests.groovy

```diff
@@ -48,25 +48,17 @@ class ChangeObjectTests extends Specification {
             }
         }
 
-        assert testInput.database.getConnection() instanceof JdbcConnection: "We cannot verify the following SQL works " +
+        def connection = testInput.database.getConnection()
```

Comment:
connection variable is used only once, you can use  "inline variable" refactoring technique.

Reply:
+